lib/mpl_toolkits/axisartist/floating_axes.py

```diff
@@ -7,7 +7,7 @@
 
 import numpy as np
 
-from matplotlib import _api, cbook
+from matplotlib import _api, cbook, transforms as mtransforms
```

Comment:
Why do we do this aliasing? From a readability point of view, this is not better. It's also not primarily about information hiding (we'd need `as _transforms` then). I know that we have it in some imports, but not consistently, others use `transforms` directly. I suspect this habit stems from times before code-aware editors where prefixes were used to carry additional information (e.g. "this is an internal module") but nowadays context information can easily provide that if I'm interested in it. IMHO the only effect of this is that it clutters the imports.

Reply:
My personal preferred approach would be to get rid of the long import list of modules at the top of most modules, import everything except backends in `matplotlib.__init__` (it may slightly slow down `import matplotlib` but in practice to get `matplotlib.axes.Axes` (and therefore plot anything) you need to import mostly everything anyways and then use `mpl.<moduke>.funcname`.  Classes, OTOH, get imported to be used without the module name.  (This is basically the style suggested by https://www.python.org/dev/peps/pep-0008/#imports.)

Reply:
Wait, WHAT? You say the modules should not define their submodule dependencies, but rely on the main package namespace to have imported everything? That seems to contradict everything I've learned and ever seen. Which part of PEP-8 are you referring to?

Reply:
Yes, that was only for the "Classes ... without the module name" part.